src/lib/server/server-match-media.ts

```diff
@@ -7,9 +7,7 @@
  */
 import {DOCUMENT} from '@angular/common';
 import {Inject, Injectable, NgZone, PLATFORM_ID} from '@angular/core';
-
-import {BreakPoint} from '../breakpoints/break-point';
-import {MatchMedia} from './match-media';
+import {BreakPoint, ɵMatchMedia as MatchMedia} from '@angular/flex-layout/core';
```

Comment:
Do we really want to import for `@angular/flex-layout` ?

Reply:
Yes, as explained offline. This is how we handle imports throughout the library due to TS conflicts if we did everything using relative paths.